docs/src/examples/components/Input/Variants/index.tsx

```diff
@@ -0,0 +1,15 @@
+import * as React from 'react'
```

Comment:
these temporary changes are introduced for the sake of example to reviewer - won't be merged